src/pages/settings/Profile/DisplayNamePage.tsx

```diff
@@ -67,6 +67,7 @@ function DisplayNamePage({currentUserPersonalDetails}: DisplayNamePageProps) {
     return (
         <ScreenWrapper
             includeSafeAreaPaddingBottom={false}
+            shouldRemoveSafePaddingBottomWhenKeyboardShown
```

Comment:
> So, that leaves us with the button padding issue

I think exactly these changes added in https://github.com/Expensify/App/pull/42143/commits/31ba00b22e78526e6e8d54866e9ed619f98b27fe were aimed to fix the problem described in https://github.com/Expensify/App/pull/42143#issuecomment-2441105627

However it's not a problem of this PR (I mean in this PR the problem becomes clearly visible, but for example after merging `edge-to-edge` [PR](https://github.com/Expensify/App/pull/52392) the same problem happens on Android right now).

Basically the fix in https://github.com/Expensify/App/pull/44761 produced these jumps now. My understanding is that we shouldn't try to fix each problem related to the keyboard in this PR, right? Do you think it would be better to revert the changes that I made and fix this "jump" problem in a separate PR?